hogben/simulate.py

```diff
@@ -26,51 +51,48 @@ def simulate_magnetic(sample, angle_times, scale=1, bkg=5e-7, dq=2,
         mp (bool): whether to simulate "minus plus" spin state.
         mm (bool): whether to simulate "minus minus" spin state.
         angle_scale (float): angle to use when scaling directbeam flux.
-        directbeam_path (str): path to directbeam file to simulate with.
+        instrument (str): path to directbeam file to simulate with.
```

Comment:
Now typing has been added to arguments, the types can be removed from the docstring, right?